Core/Frameworks/BaikalAdmin/Controller/Settings/Database.php

```diff
@@ -130,6 +141,40 @@ function validationHook(\Formal\Form $oForm, \Formal\Form\Morphology $oMorpho) {
 
                 $oForm->declareError($oMorpho->element("mysql"), $sMessage);
 
+                return;
+            }
+        } elseif (intval($oForm->modelInstance()->get("pgsql")) === 1) {
```

Comment:
That's a lot of code duplication of the mysql checks. Please try to combine both checks

Reply:
Done, checks are now combined as of 4ab75f76